guide/additional-info/changes-in-v14.md

```diff
@@ -0,0 +1,176 @@
+
+Have been removed from the `Client` in discord.js. Instead you should access these events from `Client#rest`.
+
+### Utility Changes
+
+`Util#removeMentions` has been removed, to control mentions you should use `allowedMentions` on `MessageOptions` instead.
+
+### CDN Changes
+
+Methods that return CDN URLs will now return a dynamic image URL (if available). This behavior can be overriden by setting `forceStatic` to `false` in the `MakeURLOptions` parameters.
+
+### Guild Object Changes
+
+`Guild#setRolePositions` and `Guild#setChannelPositions` have been removed. Use `RoleManager#setPositions` and `GuildChannelManager#setPositions` instead respectively.
+
+### Thread Member Manager
+
+`ThreadMemberManager#fetch` now only takes a single object of type `ThreadMemberFetchOptions`.
+
+### Role Manager
+
+`Role.comparePositions` has been removed. Use `RoleManager#comparePositions` instead.
+
+### `.deleted` Field(s) have been removed
+
+You can no longer use `#deleted` to check if a structure was deleted or not. 
+
+Check out [the issue ticket](https://github.com/discordjs/discord.js/issues/7091) for more context.
+
+### Channel changes
+
+`Channel#createAt` is now nullable. On any regular channel `#createAt` or private thread this value will always be non-null. This value is only nullable for public threads. 
+
+::: tip
+TypeScript users should narrow `Channel` types via type guards in order to get more specific typings.
+:::
+
+### Message Embed Changes
+
+- `MessageEmbed` has now been renamed to `Embed`
+
+- `#setFooter` now accepts a sole object `APIEmbedFooter` object. (add link to dapi site)
+
+- `#addField` and `#addFields` both accept an object or array of `APIEmbedField`(s) respectively. (add link to dapi site)
+
+### Message Component Changes
+
+MessageComponents have been renamed as well. They no longer have the `Message` prefix:
+
+```diff
+- const button = new MessageButton();
++ const button = new ButtonComponent();
+
+- const selectMenu = new MessageSelectMenu();
++ const selectMenu = new SelectMenuComponent();
+
+- const actionRow = new MessageActionRow();
++ const actionRow = new ActionRow();
+```
+
+Many of the analogous enums can be found be found in the discord-api-types docs (link website here)
+
+### Interaction typeguard changes
+
+The following typeguards on `Interaction` have been renamed:
+
+```diff
+- interaction.isCommand()
++ interaction.isChatInputCommand()
+
+- interaction.isContextMenu()
++ interaction.isContextMenuCommand()
+```
+
+In addition, `Interaction#isCommand`, now indicates whether the command is an *application command* or not. This differs from the previous implementation where `isCommand()` indicated if the interaction was a chat input command or not.
+
+### Voices Region Changes
+
+`VoiceRegion#vip` has been removed as the field is no longer part of the API.
+
+### Application Changes
+
+`Application#fetchAssets` has been removed as the used for this method is no longer available from the API.
```

Comment:
```suggestion
`Application#fetchAssets` has been removed as the use for this method is no longer available from the API.
```

Reply:
It would be more accurate to state:

> X has been removed as it is no longer supported by the API.

Clear and concise, whereas "the use for this method is no longer available" seems like it opens up avenues that questions its removal.